pipeline/frontend/yaml/container.go

```diff
@@ -37,7 +36,6 @@ type (
 		Tmpfs         []string               `yaml:"tmpfs,omitempty"`
 		DNS           types.Stringorslice    `yaml:"dns,omitempty"`
 		DNSSearch     types.Stringorslice    `yaml:"dns_search,omitempty"`
-		Entrypoint    types.Command          `yaml:"entrypoint,omitempty"`
```

Comment:
@6543 Why do you plan to remove the entrypoint completely?

Reply:
there was even a liter that permit the usage for normal steps ...